docs/concepts/demisto-sdk.md

```diff
@@ -0,0 +1,388 @@
+---
+id: demisto-sdk
+title: Demisto-SDK
+---
+The [Demisto-SDK](https://github.com/demisto/demisto-sdk) is a python library designed with the purpose of aiding the development process both in 
+validating the entities being developed and in assisting in the interaction between your development setup and Cortex XSOAR.
+This guide will help you get acquainted with the package, how can you install it, set it up and some 
+basic information about key commands to aid you in the development process. For additional information please see our full [Demisto-SDK documentation](https://github.com/demisto/demisto-sdk#demisto-sdk).
+
+## Installation and Setup
+
+Demisto-SDK library supports python version 3.7 and up, prior to installation please make sure you have a compatible python version installed.  
+
+To install the Demisto-SDK enter your terminal and run the command:
```

Comment:
Let's add a note that if you've followed our [bootstrap procedure](https://xsoar.pan.dev/docs/concepts/dev-setup#bootstrap) then the sdk is setup. Installing is only required if working on a repository which is not part of the Content repo.

Reply:
done